diff --git a/testsuite/SbSmallMapTests.cmake b/testsuite/SbSmallMapTests.cmake
index ce9fe6d857..baf85deceb 100644
--- a/testsuite/SbSmallMapTests.cmake
+++ b/testsuite/SbSmallMapTests.cmake
@@ -1,4 +1,5 @@
 add_executable(SbSmallMapFailureTest SbSmallMapFailureTest.cpp)
+target_link_libraries(SbSmallMapFailureTest PRIVATE Coin)
 target_compile_definitions(SbSmallMapFailureTest PRIVATE COIN_INTERNAL HAVE_CONFIG_H)
 target_include_directories(SbSmallMapFailureTest PRIVATE
   ${PROJECT_SOURCE_DIR}/src ${PROJECT_SOURCE_DIR}/include
